conf/st2.conf.sample

```diff
@@ -29,7 +29,7 @@ max_page_size = 100
 # True to mask secrets in the API responses
 mask_secrets = True
 # StackStorm API server host
-host = 127.0.0.1
```

Comment:
Does it makes sense to change the config defaults to `127.0.0.1` so we'd generate that value, instead of 0.0.0.0?